src/utils/scraping.py

```diff
@@ -50,6 +50,11 @@ def scrape_tdp_intel(cpu_name) -> float:
     :param cpu_name: CPU name, coming from get_cpu_info()
     :return: CPU Thermal Design Power (TDP) in watts
     """
+    if not isinstance(cpu_name, str):
+        raise ValueError("Input must be a string")
+    # check that the input string contains the manufacturer name
+    if not 'Intel' in cpu_name:
```

Comment:
do we want to keep this check?

Reply:
I would discard it given that: 
![image](https://github.com/MattiaSilvestri/energy-monitor/assets/86249255/477adf7c-0165-43ad-a026-5cf23a162b0b)